metaswitch/common/network_namespace.py

```diff
@@ -1,4 +1,4 @@
-# @file alarms.py
+# @file network_namespace.py
 #
```

Comment:
I know you haven't changed this file, but just having a look, the function defined uses 'signaLLing', but the very next line has 'signaLing'. This should probably be fixed up some time. not sure if this could be included, or if an issue should track it. 
@eleanor-merry thoughts?